ci/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM alpine:3.8 as conjur-test
+
+ARG KUBECTL_VERSION=1.17.0
+ARG KIND_VERSION=0.7.0
+
+RUN apk add --no-cache \
+    bash \
+    curl \
+    docker \
+    openssl \
+    wget
+
+# Install kubectl
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl && \
+    chmod +x ./kubectl && \
+    mv ./kubectl /usr/local/bin/kubectl
+
+# Install Kubernetes in Docker (kind)
+RUN curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/v${KIND_VERSION}/kind-linux-amd64 && \
+    chmod +x ./kind && \
+    mv ./kind /usr/local/bin/kind
```

Comment:
I just noticed this, but can't you just directly download the file into `/usr/local/bin`? seems like download + move is doing double the work but I can't tell why.

Reply:
This file has been deleted, but yeah, that would have made sense!